### ISSUE TITLE
[Barong v2] User activity api endpoint improvments

### ISSUE THREAD

**Issue by pkucherenk0** [openware/barong#663]
The endpoint `/api/v2/barong/resource/users/activity/topic=all` should return me info about creation of session, password changing and turning on/off of 2FA, instead we will get `"error": "No activity recorded or wrong topic "`. Same with the topic=password; topic=session; topic=otp separately. 

In case on unauthorized response is correct: ` "error": "Invalid Session"`

Barong logs:

![selection_761](https://user-images.githubusercontent.com/38254545/49230360-256adf00-f3f8-11e8-96d1-7dbd92aa999f.png)

**Comment by alinetskyi** [comment]
instead of  ``/api/v2/barong/resource/users/activity/topic=all`` you should do `` /api/v2/barong/resource/users/activity/all
``

**Comment by pkucherenk0** [comment]
Thank you, this moved me forward, but `/api/v2/barong/resource/users/activity/all` do not return me data about passwords reset and 2FA enable/disable, only session.
Calling directly to `api/v2/barong/resource/users/activity/pssword` and `api/v2/barong/resource/users/activity/otp` return me `500 Internal Server Error`.

**Comment by pkucherenk0** [comment]
The `/api/v2/barong/resource/users/activity/all` and `/api/v2/barong/resource/users/activity/session` return me session info as expected.

Right now i can see two bugs:
1. "user_ip": address is clearly not mine ip address. 
2.  "data": states same line every time and do to give any useful information, like `otp_code was incorrect while trying to log in`

Below you can see what data i get from this endpoint:

```{
        "id": 8,
        "user_id": 4,
        "user_ip": "***.**.*.*",                                   //I have replaced address with `*`
        "user_agent": "PostmanRuntime/7.3.0",
        "topic": "session",
        "action": "login::2fa",
        "result": "succeed",
        "data": "{\"note\":\"Detected suspicious browser\"}",
        "created_at": "2018-11-30T09:13:32.000Z"
    },
    {
        "id": 9,
        "user_id": 4,
        "user_ip": "***.**.*.*",                                  //I have replaced address with `*`
        "user_agent": "PostmanRuntime/7.3.0",
        "topic": "session",
        "action": "login::2fa",
        "result": "failed",
        "data": "{\"note\":\"Detected suspicious browser\"}",
        "created_at": "2018-11-30T09:14:39.000Z"
    },
    {
        "id": 8,
        "user_id": 5,
        "user_ip": "***.**.*.*",
        "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/66.0.3359.181 Chrome/66.0.3359.181 Safari/537.36",
        "topic": "session",
        "action": "login",
        "result": "failed",
        "data": null,
        "created_at": "2018-11-30T13:06:35.000Z"
    }

**Comment by ec** [comment]
Session info is only info implemented at the current moment.
We are working on another aspects of activities

**Comment by ec** [comment]
Merged in master #677
